playground-3.js

```diff
@@ -1,3 +1,23 @@
 // https://miorita.scoalaweb.ro/playgrounds/escape
 // Scrie un program care să o ajute pe Miorița să ajungă la pășune
 // Programul trebuie să funcționeze indiferent de configurația stânei.
+reset();
+
+function superMove(times = 1){
+    if (times <= 0){
+        return
+    }
+    move()
+    superMove(times - 1)
+}
+
+if (canMove()){
+    move
+}
+
+else {
+    turnRight();
+    move();
+}
+
+superMove(5);
```

Comment:
Codul tău nu funcționează pe toate scenariile. 

Uită-te, te rog, la finalul înregistrării de Marți.

Reply:
Da, nu am reusit sa rezolv pgr-3. Am o idee in mare cum ar trebui rezolvat, dar nu stiu sa o aplic inca.